### ISSUE TITLE
[PHP 8.0] implode(): Argument #2 ($array) must be of type ?array, string given

### ISSUE THREAD

**Issue by caugner** [mdsimpson/contact-form-7-to-database-extension#94]
```
TypeError: implode(): Argument #2 ($array) must be of type ?array, string given
#22 /wp-content/plugins/contact-form-7-to-database-extension/CF7DBPlugin.php(847): implode
#21 /wp-content/plugins/contact-form-7-to-database-extension/CF7DBPlugin.php(847): CF7DBPlugin::saveFormData
#20 /wp-content/plugins/contact-form-7-to-database-extension/CFDBIntegrationContactForm7.php(55): CFDBIntegrationContactForm7::saveFormData
#19 /wp-includes/class-wp-hook.php(294): WP_Hook::apply_filters
#18 /wp-includes/class-wp-hook.php(316): WP_Hook::do_action
#17 /wp-includes/plugin.php(551): do_action_ref_array
#16 /wp-content/plugins/contact-form-7/includes/submission.php(458): WPCF7_Submission::before_send_mail
#15 /wp-content/plugins/contact-form-7/includes/submission.php(81): WPCF7_Submission::proceed
#14 /wp-content/plugins/contact-form-7/includes/submission.php(25): WPCF7_Submission::get_instance
#13 /wp-content/plugins/contact-form-7/includes/contact-form.php(763): WPCF7_ContactForm::submit
#12 /wp-content/plugins/contact-form-7/includes/rest-api.php(321): wpcf7_rest_create_feedback
#11 /wp-includes/rest-api/class-wp-rest-server.php(1139): WP_REST_Server::respond_to_request
#10 /wp-includes/rest-api/class-wp-rest-server.php(985): WP_REST_Server::dispatch
#9 /wp-includes/rest-api/class-wp-rest-server.php(412): WP_REST_Server::serve_request
#8 /wp-includes/rest-api.php(354): rest_api_loaded
#7 /wp-includes/class-wp-hook.php(292): WP_Hook::apply_filters
#6 /wp-includes/class-wp-hook.php(316): WP_Hook::do_action
#5 /wp-includes/plugin.php(551): do_action_ref_array
#4 /wp-includes/class-wp.php(388): WP::parse_request
#3 /wp-includes/class-wp.php(750): WP::main
#2 /wp-includes/functions.php(1291): wp
#1 /wp-blog-header.php(16): require
#0 /index.php(17): null
```

**Comment by ammarkhan19** [comment]
Issue resolved and pull request(#96  ) created.

**Comment by caugner** [comment]
Fixed by https://github.com/mdsimpson/contact-form-7-to-database-extension/pull/96.
